js/admin.vue

```diff
@@ -52,6 +52,17 @@ export default {
                 details: this._('The error identifier is {id}', {id: e.data.event_id}),
             });
         });
+    },
+    methods: {
+        handleApiError(error) {
+            const notif = {type: 'danger', icon: 'exclamation-circle'};
+            if (error.status === 403) {
+                notif.title = this._('Operation not permitted');
+                notif.details = this._('You are not allowed to perform this operation');
```

Comment:
I'm not sure to understand how it is displayed in context?

A person could get to see this message even though no button/action lead to it?

Reply:
No, this is a generic handler (handle only 403 but it's a start and it needs improvements) that **"can"** be used on any operation without the need for a specific error handler.
This one simply display the error in the notification zone (the toast).
But this is displayed only on error on actions triggered by the user (See: https://github.com/opendatateam/udata/pull/1128/files#diff-506081ce406eac251a8b297d56b75664R129)

Note: the current situation is that most the API operation don't have error handling. This allows future bugfixes/improvements... to take advantage of this helper and/or make it handle more generic cases.